app/services/taxonomy_parser.rb

```diff
@@ -0,0 +1,14 @@
+class TaxonomyParser
+  def self.parse(content_item)
+    return [] unless content_item.key?(:links)
+
+    links = content_item.fetch(:links).deep_symbolize_keys
```

Comment:
- Failing tests ./spec/services/content_items_service_spec.rb line 24 the way the test is setup is doesn't conform well with the implementation.
  
- TaxonomyParser can be refactored to 
  ```ruby
    class TaxonomyParser
      def self.parse(content_item)
        if content_item[:links].is_a?(Hash)
          links = content_item[:links].deep_symbolize_keys
          links.fetch(:taxons, []).map do |taxon|
            taxon.fetch(:content_id)
          end
        else
          []
        end
      end
    end
  ```

- The setup of the test in ./spec/services/content_items_service_spec.rb:24 should change